Resources/Maps/_DV/Shuttles/Cargo/cargo_model_b.yml

```diff
@@ -27,7 +27,7 @@ entities:
   - uid: 1
     components:
     - type: MetaData
-      name: Cargo Shuttle
```

Comment:
file name is "Model B", this says "B Model"
Should probably change both to be "Model B" for consistency